src/math/Average.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+namespace NuTo
+{
+
+//! @brief calculates the cumulative moving average
+struct Average
```

Comment:
Dunno. I wrote it to calculate the average number of newton iterations. Say, if you run newmark and want to know the average number of newton raphson iterations per time step. IMO, it doesnt hurt.